aws/resource_aws_mq_broker.go

```diff
@@ -217,8 +222,17 @@ func resourceAwsMqBroker() *schema.Resource {
 							Optional: true,
 						},
 						"audit": {
-							Type:     schema.TypeBool,
-							Optional: true,
+							Type:         nullable.TypeNullableBool,
+							Optional:     true,
+							ValidateFunc: nullable.ValidateTypeStringNullableBool,
+							DiffSuppressFunc: func(k, o, n string, d *schema.ResourceData) bool {
```

Comment:
This seems like something that could be useful in other places where we would be using nullable bools - Do you want to extract it to `diff_suppress_funcs.go` or somewhere similar?